_posts/2023-12-24-update.md

```diff
@@ -0,0 +1,52 @@
+
+## Release Schedule
+
+**Next Deadline: 1.30 Cycle Begins, January 2024**
+
+We are in the holiday period between releases; dates for 1.30 have not been published yet. If you want to become a part of the [Release team](https://github.com/kubernetes/sig-release/issues/2377), apply soon! Shadow applications are open until Wednesday, January 3rd 2024 and will be reviewed until January 12th 2024. Use this [form](https://docs.google.com/forms/d/e/1FAIpQLScTWKYMqO8TUKjqqvQzmjZ1y03mefrKqrAiPvhVqkvdzVrHAQ/viewform) to apply. 
+
+## Featured PRs
+
+
+## KEP of the Week
+
+### [KEP 4020: Unknown Version Interoperability Proxy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/4020-unknown-version-interoperability-proxy)
+
+This KEP proposes to solve an issue that arises when multiple apiservers with different versions are running in a cluster. All the different apiservers might not be able to server all the resources in all the different versions. This can happen when a cluster upgrade/downgrade is going or when a runtime-config changes. This proposed filter will proxy clients to an apiserver that is capable of handling the request from the client. The KEP proposes to use the existing `StorageVersion` API to figure out which group, version and resources an apiserver can serve. If the apiserver doesn't have the requested group/version/resource locally, the request will be proxied to one of the apiservers listed in the `ServerStorageVersion` object. 
+
+This KEP was in alpha stage in the v1.28 release
+
+## Other Merges
+
+* Replace k8s.io/utils/integer functions with [built-in max and min functions](https://github.com/kubernetes/kubernetes/pull/122310).
+* Clarification to warning around [duplicate env var entries](https://github.com/kubernetes/kubernetes/pull/122126) to mention that duplicates can be dropped when using apply.
+* Clean up of [unused HealthcheckType](https://github.com/kubernetes/kubernetes/pull/122083).
+* [Scheduler First Fit](https://github.com/kubernetes/kubernetes/pull/122435) has been fixed after it was [reverted](https://github.com/kubernetes/kubernetes/pull/122317) due to a flaky test.
+* Scheduler configuration validation gets a [more understandable error message](https://github.com/kubernetes/kubernetes/pull/122387).
+* kubectl to [initialize the customizable attachFunc](https://github.com/kubernetes/kubernetes/pull/122447) when the `--attach` flag is passed to fix a bug that was causing the initialization to be skipped.
+* Update to the [help message of the interactive flag in kubectl delete](https://github.com/kubernetes/kubernetes/pull/122441).
+
+## Deprecated
+
+* GA FeatureGates being removed: [APISelfSubjectReview](https://github.com/kubernetes/kubernetes/pull/122032)
```

Comment:
```suggestion
* GA feature gates being removed: [APISelfSubjectReview](https://github.com/kubernetes/kubernetes/pull/122032)
```

But also, this isn't a deprecation. It's a removal, which is different. Deprecated things still work, you should stop using them. Removed things aren't there for you to try using them.